src/main/java/athena/Parser.java

```diff
@@ -1,5 +1,7 @@
 package athena;
 
+import athena.commands.*;
```

Comment:
Prof said cannot using `*` to import. Refer to https://github.com/nus-cs2113-AY2021S1/forum/issues/36